Add 5.1 features to administration area (#1174)
contao/docs: docs/manual/administration-area/accessing-and-structure-of-the-backend.en.md

**Homepage:** A click on the Contao logo takes you back to the homepage of the back end.

> **Manual:** Clicking on this link opens the manual.

Reviewer: ```suggestion
**Manual:** Clicking this link opens the manual.
```

> **Save Favorite:** {{< version-tag "5.1" >}} By clicking on this link you can save the current backend URL as a favorite entry. Existing 

Reviewer: ```suggestion
**Save Favorite:** {{< version-tag "5.1" >}} By clicking on this link you can save the current back end URL as a favorite entry. Existing 
```

favorite links are listed in the navigation area.

> **Notes:** Clicking on this link opens a modal and displays possible hints (e.g. maintenance mode).

Reviewer: ```suggestion
**Notes:** Clicking this link opens a modal and displays possible hints (e.g. maintenance mode).
```

**Design:** {{< version-tag "5.1" >}} You can choose between light and dark mode.

> **Debug Mode:** {{< version-tag "4.8" >}} Clicking on this link will switch the [debug mode](/en/system/debug-mode/) on or off.

Reviewer: ```suggestion
**Debug Mode:** {{< version-tag "4.8" >}} Clicking this link will switch the [debug mode](/en/system/debug-mode/) on or off.
```
